lib/normalize-query-options/order.js

```diff
@@ -1,8 +1,8 @@
+const { normalizeArray } = require('./helpers')
+
 function normalizeOrder (options) {
   const order = options.order || options.orderByFields
-  if (!order) return
```

Comment:
There is a minor difference there. If an empty string is given, the old code returns an `undefined` and the new code returns an empty array. Does it matter?